### ISSUE TITLE
STPAddCardViewController issue

### ISSUE THREAD

**Issue by apisit** [stripe/stripe-ios#359]
1. When error appears in STPAddCardViewController. e.g. card declined.
The loading indicator in rightBarButtonItem never stop spinning and Cancel button is unable to tap. User is unable to close view controller.

2. Unable to set prefilledInformation to STPAddCardViewController. (I tried setting prefilledInformation both outside and inside presentViewController completion )

```
let addCardViewController = STPAddCardViewController()
addCardViewController.delegate = self
let navigationController = UINavigationController(rootViewController: addCardViewController)
self.presentViewController(navigationController, animated: true, completion: {
                let info = STPUserInformation()
                info.email = "test@email.com"
                addCardViewController.prefilledInformation = info
})
```

Am I doing it wrong or missing something?

Thanks.

**Comment by DmitryBulaienko** [comment]
+1, got same issues

**Comment by adamshovav** [comment]
+1. got same issue with forever-spinning indicator. 
@apisit , you should specify prefiled information just before presenting controller, not in the completion

**Comment by benzguo** [comment]
@apisit @DmitryBulaienko – thanks for reporting! The STPAddCardViewController issue has been fixed in the latest release, [v8.0.1](https://github.com/stripe/stripe-ios/releases/tag/v8.0.1).
